doc/faq.rst

```diff
@@ -280,6 +280,19 @@ program: Insert the following instructions in your main script::
 You can find more default on the new start methods in the `multiprocessing
 documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods>`_.
 
+Why does my job use more cores than specified with n_jobs under OSX or Linux?
+-----------------------------------------------------------------------------
+This can happen when the parallelized computation is handled outside of Python by another library,
```

Comment:
Please try to keep lines under 80 characters

Reply:
Thanks. I'll refactor the text.